indico/cli/i18n.py

```diff
@@ -113,62 +244,246 @@ def _wrapper(**kwargs):
     return _wrapper
 
 
-def _make_command(cmd_name):
+def _run_command(cmd_name, extra=None):
     cmd_class = getattr(frontend, re.sub(r'_(js|react)$', '', cmd_name))
-    cmd = click.command(cmd_name.replace('_', '-'))(wrap_distutils_command(cmd_class))
-    for opt, short_opt, description in cmd_class.user_options:
-        long_opt_name = opt.rstrip('=')
-        var_name = long_opt_name.replace('-', '_')
-        opts = ['--' + long_opt_name]
-
-        if short_opt:
-            opts.append('-' + short_opt)
-
-        default = DEFAULT_OPTIONS.get(cmd_name, {}).get(var_name)
-        is_flag = not opt.endswith('=')
-        cmd = click.option(*(opts + [var_name]), is_flag=is_flag, default=default, help=description)(cmd)
-    return cmd
+    options = DEFAULT_OPTIONS.get(cmd_name, {})
+    if extra:
+        options.update(extra)
+    if plugin_dir := options.get('plugin_dir', None):
```

Comment:
```suggestion
    if plugin_dir := options.get('plugin_dir'):
```

this is already the default